<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIListAccount</class>
 <widget class="QMainWindow" name="UIListAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestion des comptes - Gestionnaire : </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>37</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayoutNorth">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTitle">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Les comptes du client : </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerLeft">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_Tableview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>581</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayoutTableview">
     <item>
      <widget class="QTableView" name="tableViewAccounts">
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_Buttons">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>581</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <item>
      <spacer name="horizontalSpacerButtons">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonNouveau">
       <property name="text">
        <string>Nouveau</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonModifier">
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOuvrir">
       <property name="text">
        <string>Ouvrir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonGeler">
       <property name="text">
        <string>Geler</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonActiver">
       <property name="text">
        <string>Activer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Fermer</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
